packages/Interface/src/DTK_C_API.h

```diff
@@ -0,0 +1,209 @@
+/****************************************************************************
+ * Copyright (c) 2012-2017 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ ****************************************************************************/
+#ifndef DTK_C_API_H
+#define DTK_C_API_H
+
+#include <DataTransferKit_config.hpp>
+
+#include <DTK_Types.h>
+
+#ifndef __cplusplus
+#include <stdbool.h> // for bool
+#include <stddef.h>  // for size_t
+#else
+#include <cstddef> // for size_t
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*! Return current DTK version. */
+extern const char *DTK_version();
```

Comment:
The * is put in a strange position. I think it would be worth to use `// clang-format off`. Same below.